src/util/branch-name.ts

```diff
@@ -103,7 +103,22 @@ const AUTORELEASE_PATTERN =
   /^release-?(?<component>[\w-.]*)?-v(?<version>[0-9].*)$/;
 class AutoreleaseBranchName extends BranchName {
   static matches(branchName: string): boolean {
-    return !!branchName.match(AUTORELEASE_PATTERN);
+    const match = branchName.match(AUTORELEASE_PATTERN);
```

Comment:
If I'm understanding the problem properly, could this safely be simplified to:

```
const BRANCH_PREFIX= 'release-please--branches';
if (branchName.startsWith(BRANCH_PREFIX)) return false;
```

Reply:
Ah, yes, that works.  I'll do that.